### PR TITLE
Initial FE commit

### PR DESCRIPTION
**Pull request by maks-hayduk**

### CODE REVIEW

File: Src/FE/public/index.html [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+      Learn how to configure a non-root public URL by running `npm run build`.
+    -->
+    <title>React App</title>
+  </head>
+  <body>
+    <noscript>You need to enable JavaScript to run this app.</noscript>
+    <div id="root"></div>
+    <!--
+      This HTML file is a template.
+      If you open it directly in the browser, you will see an empty page.
```
Comment (Ostapcs) [inline Src/FE/public/index.html:29]: remove comments

File: Src/FE/src/App.tsx [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+const App: React.FC = () => {
+  return (
+   <h1>heya</h1>
```
Comment (Ostapcs) [inline Src/FE/src/App.tsx:5]: ?????

File: Src/FE/public/index.html [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+    <!--
+      Notice the use of %PUBLIC_URL% in the tags above.
+      It will be replaced with the URL of the `public` folder during the build.
+      Only files inside the `public` folder can be referenced from the HTML.
+
+      Unlike "/favicon.ico" or "favicon.ico", "%PUBLIC_URL%/favicon.ico" will
+      work correctly both with client-side routing and a non-root public URL.
+      Learn how to configure a non-root public URL by running `npm run build`.
+    -->
+    <title>React App</title>
```
Comment (Ostapcs) [inline Src/FE/public/index.html:22]: add title for our app

**Review by Ostapcs — APPROVED** [review]

**Review by savant48 — APPROVED** [review]
О_о

**Pull request merged**